src/lib.rs

```diff
@@ -582,6 +582,32 @@ impl Operand {
         )
     }
 
+    /// Create an `Operand` that designates a use of a vreg and places
+    /// no constraints on its location (i.e., it can be allocated into
+    /// either a register or on the stack).
+    #[inline(always)]
+    pub fn any_use(vreg: VReg) -> Self {
+        Operand::new(
+            vreg,
+            OperandConstraint::Any,
+            OperandKind::Use,
+            OperandPos::Early,
+        )
+    }
+
+    /// Create an `Operand` that designates a def of a vreg and places
+    /// no constraints on its location (i.e., it can be allocated into
+    /// either a register or on the stack).
+    #[inline(always)]
```

Comment:
Do these _really_ need to be `inline(always)`? That will force inlining even in debug builds, which has a negative impact on compile times. Is plain `inline` good enough here?

Reply:
IIRC this was actually a hotspot when running debug builds at one point. The constructors are bitwise ORs/shifts to put together bitfields, and in most cases have constants for all but vreg, so turn into `vreg | 0xconstant`. that's *probably* faster to compile than a callsite, though I haven't measured. In any case, I'm following the pattern for other ctors here on these new ones, and am hesitant to touch load-bearing perf things without much more testing :-)

Reply:
(I'll go ahead and merge now with this but we can definitely talk more about inlining and compile time if you think it's warranted)